packages/mdc-rtl/README.md

```diff
@@ -28,76 +26,20 @@ npm install --save @material/rtl
 
 ## Usage
 
-Simply `@import "@material/rtl/mixins";` and start using the mixins. Each mixin is described below.
+### Sass Mixins
 
-### mdc-rtl
+`$base-property` is a base box-model property. e.g. margin / border / padding.
+`$pos` is a horizontal position property, either "left" or "right".
```

Comment:
Sorry...but I think maybe you're look at an old commit. I changed this in the latest commit.

Or maybe I screwed something up in my commits...